frontend/src/components/AddTask.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { Button, TextField, Typography } from '@material-ui/core';
+
+const AddItem = props => {
+    const [name, setName] = useState('');
+
+    const { handleAddTask } = props;
```

Comment:
you can destructure props above, so `const AddItem = ({handleAddTask}) => ...`, same in other components

Reply:
done